### ISSUE TITLE
Closing Neotree and reopening it will sometimes error (and fail to open)

### ISSUE THREAD

**Issue by miversen33** [miversen33/netman.nvim#99]
Closing Neotree `remote`, and then reopening it will sometimes produce the following error
```
E5108: Error executing lua ...im/site/pack/packer/start/nui.nvim/lua/nui/tree/init.lua:254: Expected Lua number
stack traceback:
        [C]: in function 'nvim_win_get_cursor'
        ...im/site/pack/packer/start/nui.nvim/lua/nui/tree/init.lua:254: in function 'get_node'
        ...packer/start/netman.nvim/lua/netman/ui/neo-tree/init.lua:315: in function 'navigate'
        ...ker/start/neo-tree.nvim/lua/neo-tree/sources/manager.lua:451: in function 'navigate'
        ...packer/start/neo-tree.nvim/lua/neo-tree/command/init.lua:174: in function 'do_show_or_focus'
        ...packer/start/neo-tree.nvim/lua/neo-tree/command/init.lua:132: in function 'execute'
        ...packer/start/neo-tree.nvim/lua/neo-tree/command/init.lua:140: in function '_command'
        [string ":lua"]:1: in main chunk
```

Not really sure what is causing this, and fun fact, dumping logs is broken :)

**Comment by miversen33** [comment]
[logs.txt](https://github.com/miversen33/netman.nvim/files/10155967/netman-logs.txt)

**Comment by miversen33** [comment]
Closed with merge of #101

**Comment by miversen33** [comment]
Whoops, wrong issue :upside_down_face:

**Comment by miversen33** [comment]
Closed per #104
